pom.xml

```diff
@@ -101,10 +101,11 @@
         <hikaricp.version>3.4.5</hikaricp.version>
         <h2.version>1.4.200</h2.version>
         <snowflake.version>3.13.5</snowflake.version>
+        <redshift.version>2.0.0.4</redshift.version>
 
 
         <!-- Legend -->
-        <legend.pure.version>1.30.0</legend.pure.version>
+        <legend.pure.version>1.29.1-SNAPSHOT</legend.pure.version>
```

Comment:
We don't want to commit this

Reply:
We actually don't now. I have removed it